programs/ziti-edge-tunnel/package/CPackGenConfig.cmake

```diff
@@ -1,12 +1,12 @@
 cmake_policy(SET CMP0057 NEW)
 
 if(CPACK_GENERATOR MATCHES "RPM")
-    set(CPACK_RPM_BUILDREQUIRES "cmake >= ${CMAKE_MINIMUM_REQUIRED_VERSION}, systemd-devel, gawk, gcc-c++ >= 4.9, python3, openssl-devel, zlib-devel")
+    set(CPACK_RPM_BUILDREQUIRES "cmake >= ${CMAKE_MINIMUM_REQUIRED_VERSION}, systemd-devel, gawk, gcc-c++ >= 4.9, python3, openssl-devel, zlib-devel, libacl-devel")
     if(CPACK_OS_RELEASE_NAME IN_LIST CPACK_RPM_DISTRIBUTIONS AND CPACK_OS_RELEASE_VERSION VERSION_GREATER "7")
         list(APPEND CPACK_RPM_BUILDREQUIRES "systemd-rpm-macros")
     endif()
     set(CPACK_RPM_PACKAGE_SOURCES OFF)
-    set(CPACK_RPM_PACKAGE_REQUIRES "iproute, gawk, systemd, libatomic, openssl-libs, zlib, polkit")
+    set(CPACK_RPM_PACKAGE_REQUIRES "iproute, gawk, systemd, libatomic, openssl-libs, zlib, polkit, libcap")
```

Comment:
no runtime dependency on `libacl`?

Reply:
The answer here is related to the question above. In the case of static compilation, there is no other runtime dependency.

```
ldd build/programs/ziti-edge-tunnel/Debug/ziti-edge-tunnel
        linux-vdso.so.1 (0x00007ffd01f5a000)
        libcap.so.2 => /lib64/libcap.so.2 (0x00007f4efe7dc000)
        libz.so.1 => /lib64/libz.so.1 (0x00007f4efe7c2000)
        libatomic.so.1 => /lib64/libatomic.so.1 (0x00007f4efe7b8000)
        libm.so.6 => /lib64/libm.so.6 (0x00007f4efe6d7000)
        libresolv.so.2 => /lib64/libresolv.so.2 (0x00007f4efe6c5000)
        libc.so.6 => /lib64/libc.so.6 (0x00007f4efe4e5000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f4efe7fd000)
```

Reply:
I think it's generally preferable to link with shared libraries when they are available though, unless there's a reason not to (e.g. GH release builds). We manage currently this (.so's in package binaries, .a's in GH binaries) by using a special cmake preset (ci-linux-static-libssl, which should be renamed) for the GH release build.

If we're using the static libacl.a from vcpkg then I would not think we need `libacl-devel` in `CPACK_RPM_BUILDREQUIRES`?

Reply:
The reason here was to minimize unknowns in library versioning and implementations, specifically, when it came to OpenWRT builds (at least for common architectures). Additionally, if I remember correctly, older versions of libacl didn't ship a pkg-config file, which meant that you couldn't leverage the pkg-config cmake module used here. The port addresses both those issues. Correct, I'll remove it. That was probably cruft from before the switch.

Reply:
Ok. this is the main reason why there are distro-specific subdirectories in vcpkg-overlays/linux-syslibs.